packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -36,27 +41,50 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private handleImageError() {
+    let message =
+      'The image url is not valid. You might want to add a "fallback" property.';
+
+    if (this.fallback) {
+      message = 'The image url is not valid or could not be loaded.';
+      this.host.querySelector('img')!.src = this.fallback;
+    }
+    this.bindings.engine.logger.error(message, this.host);
+  }
+
   public render() {
-    const url = this.url;
+    let url = this.url;
 
     if (!url) {
-      this.host.remove();
-      return;
+      if (this.fallback) {
```

Comment:
Should we validate that fallback url is reachable and doesn't result in the same error?

Reply:
We could but it would require a lot of nesting. I would need to validate in every edge case where I need the fallback if the fallback is valid. I think users will most likely use a local image file path so it won't be a problem for them.